src/App.svelte

```diff
@@ -121,678 +182,143 @@
-            >
-              Stephen Mount
-            </h1>
-            <p
-              class="font-light text-xs sm:text-sm md:text-sm lg:text-sm xl:text-sm py-1 font-normal hidden md:block lg:block break-words"
-            >
-              After 10+ years in software engineering, I am coming up with
-              ideas and things to reverse engineer things for the new wave
-              of ARM etc etc
-            </p>
-            <div
-              class="hidden lg:inline-flex md:inline-flex flex flex-inline"
-            >
-              <div class="flex flex-inline">
-                <a target="_blank" href="https://github.com/stemount">
-                  <i
-                    class="fa fa-github-alt pr-1 pt-6"
-                    style="
-                      font-size: 18px;
-                      color: rgb(109, 71, 199);
-                      font-style: normal;
-                    "
-                  ></i>
-                  <p
-                    class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
-                  >
-                    GitHub
-                  </p>
-                </a>
-              </div>
-              <div class="flex flex-inline">
-                <a target="_blank" href="https://ste.london">
-                  <i
-                    class="fa fa-link pr-1 pt-6 text-purple-500"
-                    style="
-                      font-size: 18px;
-                      color: rgb(109, 71, 199);
-                      font-style: normal;
-                    "
-                  ></i>
-                  <p
-                    class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
-                  >
-                    Website
-                  </p>
-                </a>
-              </div>
-            </div>
-            <!-- for small screens -->
-            <div
-              class="flex-inline lg:hidden sm:flex-inline md:hidden flex flex-inline"
-            >
-              <div>
-                <a target="_blank" href="https://github.com/stemount">
-                  <i
-                    class="fa fa-github-alt pt-5 pr-5"
-                    style="font-size: 20px; color: rgb(109, 71, 199)"
-                  ></i>
-                </a>
-              </div>
-              <div>
-                <a target="_blank" href="https://ste.london">
-                  <i
-                    class="fa fa-link pt-5 pr-5"
-                    style="font-size: 20px; color: rgb(109, 71, 199)"
-                  ></i>
-                </a>
-              </div>
-            </div>
-          </div>
+<!--Sveltified Version Below -->
+<!--DATA STRUCTURE
+ {
+      name: "EddieBot",
+      imageSrc: "https://user-images.githubusercontent.com/624760/91444835-3797dc80-e86d-11ea-8dfa-d7993a4d382d.png",
+      description: "Discord bot for Eddie Jaoude's Discord server.",
+      link: "https://github.com/EddieJaoudeCommunity/EddieBot",
+  }
+  -->
```

Comment:
Maybe we can delete these comments, what do you think @UnevenCoder ?

Reply:
Yup 100% the goal was to just check if everything is well then I can work on the responsiveness later tommorow 😀

Reply:
awesome thanks! If you need any help with the responsiveness let us know 😃

Reply:
Will do .👍